tests/run_all.sh

```diff
@@ -2,22 +2,23 @@
 
 export TEST_DIR="$PWD/tests"
 tests_to_run=(
```

Comment:
I'm personally a fan of being very explicit with tests, but we could do test discovery now.